ui/src/routes/results.tsx

```diff
@@ -101,6 +102,14 @@ function Results() {
 
   return (
     <Layout className="layout">
+      <Helmet>
+        <meta
+          name="description"
+          content={
+            "The CERN Academic Training lectures cover physics and technology research results, as well as leading-edge news from other disciplines. Past lectures often present a great historical value."
```

Comment:
could you please add the text in a constant, since we are using it in many places